src/org/labkey/test/tests/visualization/ColumnChartTest.java

```diff
@@ -83,12 +83,8 @@ private void doSetup()
         selectQuery("study", DATA_SOURCE_1);
         click(Locator.linkWithText("edit definition"));
 
-        waitForText("Edit Dataset Definition");
-
         log("Set the '" + PREGNANCY_COLUMN_NAME + "', '" + LANGUAGE_COLUMN_NAME + "', and '" + SIGNATURE_COLUMN_NAME + "' fields to be dimensions but not measures.");
-
         DatasetDesignerPage datasetDesignerPage = new DatasetDesignerPage(getDriver());
```

Comment:
This should probably wait for a page load. The `waitForText` was compensating for its absence and this might cause intermittent race conditions (depending how much the designer page waits for things). Might as well use the page class while we're at it.
```suggestion
        DatasetDesignerPage datasetDesignerPage = new DatasetPropertiesPage(getDriver()).clickEditDefinition();
```

Reply:
good call, I like it

Reply:
For context, we're clicking 'Edit Definition' from a query-begin.view page in schema browser- do we have a page class for that?  Would be nice to have pages for those

Reply:
Ahh, my bad. I just saw the "edit definition" and assumed it was on the dataset designer.
As for a schema browser page class, agreed. Currently, most of the functionality is sitting in `BaseWebDriverTest` (`selectSchema`, `selectQuery`, etc.).